test/expected/sg_tg_multiple_tf_separate/test-vpc3.tf

```diff
@@ -2,5 +2,5 @@
 resource "ibm_is_security_group" "test-vpc3--vsi0-subnet30" {
   name           = "sg-test-vpc3--vsi0-subnet30"
   resource_group = local.sg_synth_resource_group_id
-  vpc            = local.sg_synth_test-vpc3_id
+  vpc            = local.sg_synth_test-vpc3--vsi0-subnet30_id
```

Comment:
Is this correct? VPC name includes vsi and subnet?

Reply:
fixed